### PR TITLE
CASMINST-5746

### PR DESCRIPTION
**Pull request by atifsyedali**
## Summary and Scope

IUF operator should conditionally include operations based IUF product manifest contents

_Is this change backwards incompatible, backwards compatible, or a backwards compatible bugfix?_

Yes

## Issues and Related PRs

_List and characterize relationship to Jira/Github issues and other pull requests. Be sure to list dependencies._

* Resolves [CASMINST-5746](https://jira-pro.its.hpecorp.net:8443/browse/CASMINST-5746)

## Testing

_List the environments in which these changes were tested._

frigg

## Pull Request Checklist

- [x] Version number(s) incremented, if applicable
- [x] Copyrights updated
- [x] License file intact
- [x] Target branch correct
- [ ] CHANGELOG.md updated
- [ ] Testing is appropriate and complete, if applicable
- [ ] [HPC Product Announcement](https://cray.slack.com/archives/C026TVCSXLH) prepared, if applicable

### CODE REVIEW

File: charts/v2.0/cray-nls/values.yaml [hunk 1]
```diff
@@ -104,7 +104,7 @@ cray-service:
       name: "cray-nls"
       image:
         repository: artifactory.algol60.net/csm-docker/stable/cray-nls
-        tag: 0.9.18
+        tag: 0.9.19
```
Comment (haasken-hpe) [inline charts/v2.0/cray-nls/values.yaml:107]: Can we actually bump this to 0.9.20 to include Jesse's change? I'm just about to merge the PR that does the version bump here: https://github.com/Cray-HPE/cray-nls/pull/152

Then I'll push the 0.9.20 tag to the cray-nls repo. I can let you know when that's done.

Comment (haasken-hpe) [reply charts/v2.0/cray-nls/values.yaml:107]: I merged that PR and pushed the v0.9.20 tag. The build of the container image is occurring here: https://github.com/Cray-HPE/cray-nls/actions/runs/4086357442

**Review by haasken-hpe — APPROVED** [review]

File: charts/v2.0/cray-nls/Chart.yaml [hunk 2]
```diff
@@ -24,7 +24,7 @@
 
 apiVersion: v2
 name: "cray-nls"
-version: 1.4.50
+version: 1.4.52
```
Comment (haasken-hpe) [inline charts/v2.0/cray-nls/Chart.yaml:27]: Probably don't really need to skip 1.4.51, but not a big deal.

Comment (atifsyedali) [reply charts/v2.0/cray-nls/Chart.yaml:27]: I think I had an intermediate debug version which may have produced an artifact. Just making sure we don't hit issues with charts not being overwritten

Comment (atifsyedali) [reply charts/v2.0/cray-nls/Chart.yaml:27]: Actually someone else did 51.

**Pull request merged**